### ISSUE TITLE
Feature Request: Function for loading MF via Code

### ISSUE THREAD

**Issue by manfredsteyer** [picardjs/picard#4]
We've already discussed this. This is just to have an official record here: It would be great if we could load an MF via a function, e.g. as Angular's router uses Lambda-Expressions for lazy loading (e.g. `loadComponent: () => loadMF(...)`)

**Comment by FlorianRappl** [comment]
Yes totally, right now the way would be I think via `loadComponent`, but this would / could introduce the life cycle (and other things). There should be a better way.

Will be added in the 0.1.0 release. Thanks for keeping track of this!

**Comment by FlorianRappl** [comment]
Right now this is named `getExport` and can be found in the scope (i.e., the `picard` property of Window in the browser).

```js
// with discovery service
picard.getExport({ name: 'routes', source: 'name-of-mf' })

// direct / without discovery service
picard.getExport({ name: 'routes', source: 'URL-of-js', kind: 'module', remoteName: 'foo', remoteType: 'esm' })
```

Shipped with 0.0.7 ( part of initial release 0.1.0 ).
